### ISSUE TITLE
Output probabilities for multi-class classification problems exceed the range of [0, 1]

### ISSUE THREAD

**Issue by theray33** [ThilinaRajapakse/simpletransformers#694]
![model_outputs](https://user-images.githubusercontent.com/69350383/92301851-cfd84480-ef84-11ea-86cc-3504f01b35c2.png)

Hi @ThilinaRajapakse , when using model.predict in multi-class classification problem, the model outputs are not within the range of [0, 1] as expected from softmax layer.

**Comment by ThilinaRajapakse** [comment]
Model outputs do not have a softmax applied to them. It's the raw logits from the model.
